ui/src/components/Equipment/GenericEquipment.jsx

```diff
@@ -1,57 +1,28 @@
 import React from 'react';
 import { Row, Col } from 'react-bootstrap';
 import Collapsible from 'react-collapsible';
-import { BsChevronUp, BsChevronDown } from 'react-icons/bs';
+import { BsChevronDown } from 'react-icons/bs';
 import EquipmentState from './EquipmentState';
-import styles from './genericEquipmentControl.module.css';
+import styles from './GenericEquipmentControl.module.css';
 
 export default function GenericEquipment(props) {
-  function renderCollapsibleHeaderOpen(cssClass) {
-    return <BsChevronUp className={cssClass} size="1em" />;
-  }
-
-  function renderCollapsibleHeaderClose(cssClass) {
-    return <BsChevronDown className={cssClass} size="1em" />;
-  }
-
-  function renderEquipmentState() {
-    return (
-      <EquipmentState
-        state={props.state}
-        equipmentName={props.name}
-        style={{
-          margin: 0,
-          width: 'inherit',
-          borderBottomLeftRadius: '0%',
-          borderBottomRightRadius: '0%',
-        }}
-      />
-    );
-  }
+  const { name, state, initialOpen, children } = props;
 
   return (
     <Row className="mb-3">
-      <Col sm={12} className={styles.generic_equipment_container}>
+      <Col sm={12} className={styles.container}>
         <Collapsible
-          open={props.CollapseOpen}
+          triggerClassName={styles.trigger}
+          triggerOpenedClassName={styles.trigger}
+          open={initialOpen}
           trigger={
-            <div>
-              {' '}
-              {renderEquipmentState()}{' '}
-              {renderCollapsibleHeaderClose(styles.generic_equipment_arrow_p)}
-            </div>
-          }
-          triggerWhenOpen={
-            <div>
-              {' '}
-              {renderEquipmentState()}{' '}
-              {renderCollapsibleHeaderOpen(styles.generic_equipment_arrow_p)}
-            </div>
```

Comment:
Instead of rendering two different triggers wen open/closed, I now use the `aria-expanded` attribute to rotate the chevron icon.

Reply:
Ah, next and `aria-expanded` is handed by `bootstrap` then I assume, so the only thing you need to do is to add a style ?

Reply:
Indeed. Could have used other solutions too, like a different class when opened, one of the default global classes provided by `react-collapsible`, etc. But yeah the idea is to rotate the icon with CSS instead of rendering two different icons.